Troubleshooting.md

```diff
@@ -0,0 +1,210 @@
+We've listed all the known issues with the CORTX software and the steps to resolve them below. Please refer to our [support documentation](SUPPORT.md) to report new issues or want us to document steps to resolve an issue. 
+
+# Known Issues
+
+<details>
+<summary>Firmware bundle upload and update functionality not working</summary>
+<p>
+
+### Symptoms and causes
+  
+  1. Failed to fetch update status.
+  2. Failed to upload firmware bundle file.
+  3. Failed to update the firmware.
+
+### Resolution
+
+  1. Failed to fetch update status.
+    a. Refresh CMP UI after 30 secs.
```

Comment:
I have no idea what any of this means.  Nor will the reader.  This is not a useful document yet.